src/Adapters/Files/FilesAdapter.js

```diff
@@ -56,6 +59,37 @@ export class FilesAdapter {
    * @return {string} Absolute URL
    */
   getFileLocation(config: Config, filename: string): string {}
+
+  /** Validate a filename for this adaptor type (optional)1G
+   *
+   * @param {string} filename
+   *
+   * @returns {null|*|Parse.Error} null if there are no errors
```

Comment:
@returns {null|Parse.Error} I don't know what the asterisk means.

Reply:
Intellij being over-zealous